.github/workflows/terraform-apply.yaml

```diff
@@ -24,17 +24,50 @@ jobs:
         matrix:
           repository:
             # The naming convention of the files is $REPO_$BRANCH.tfvars
+            - bootstack-actions_main
```

Comment:
Bootstack-actions will be deprecated. But it's not a big issue to apply to this project too.  I am fine with that.

Reply:
Thanks :)